DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -633,6 +636,15 @@ function Setup-Secrets {
 
     $secretName = $prefix + "datax-sa-" + $sparkBlobAccountName    
     Setup-Secret -VaultName $vaultName -SecretName $secretName -Value $tValue
+	
+	if($setupAdditionalResourcesForSample -eq 'y'){
```

Comment:
Better to deploy all sample related stuffs in deploySample.ps1. Any reason why you have this here?

Reply:
moved to deploySample.ps1